src/processTargets/modifiers/scopeHandlers/NestedScopeHandler.ts

```diff
@@ -21,34 +21,45 @@ import type { ScopeHandler } from "./scopeHandler.types";
 export default abstract class NestedScopeHandler implements ScopeHandler {
   public abstract readonly iterationScopeType: ScopeType;
 
+  /**
+   * We expand to this scope type before looking for instances of the scope type
+   * handled by this scope handler.  In most cases the iteration scope will
+   * suffice, but in some cases you want them to diverge.  For example, you
+   * might want the default iteration scope to be `"file"`, but you don't need
+   * to expand to the file just to find instances of the given scope type.
+   */
+  protected get searchScopeType(): ScopeType {
```

Comment:
I added this in preparation for `"next air"`, which will use `line` as its search scope, but `document` as its iteration scope